components/badges.css

```diff
@@ -0,0 +1,72 @@
+.badge-wrapper {
+  position: relative;
+  box-sizing: border-box;
+}
+.badge-wrapper i {
+  display: block;
+  font-size: 3rem;
+}
+.badge-wrapper button {
+  padding: 1rem;
+  border: none;
+  border-radius: 5px;
+  color: var(--hover-bg);
+  background-color: var(--secondary-color);
+}
+.badge-icon-val {
+  position: absolute;
+  width: 1rem;
+  top: -1rem;
+  right: -1rem;
+  padding: 0.5rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 50%;
+  color: var(--hover-bg);
+  font-weight: 700;
+  background-color: var(--success-color);
+}
+.badge-btn-val {
+  position: absolute;
+  width: 1rem;
+  font-weight: 700;
+
+  top: -1rem;
+  right: -3rem;
+  padding: 0.5rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 50%;
+  color: var(--hover-bg);
+  background-color: var(--success-color);
+}
+.badge-img {
+  position: absolute;
+  width: 1rem;
+  top: 0;
+  right: 0;
+  padding: 0.5rem;
+  font-weight: 700;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 50%;
+  color: var(--hover-bg);
+  background-color: var(--success-color);
+}
+.badge-stat {
+  position: absolute;
+  right: 1rem;
+  bottom: 1rem;
+  padding: 0.5rem;
+  font-weight: 700;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 50%;
+  border: 4px solid var(--hover-bg);
+  color: var(--hover-bg);
+  background-color: var(--success-color);
+}
```

Comment:
I see many properties are repeated, can't anything be done about that? Can you make a common class to do that?

Reply:
@horsemaker thank you for pointing it out. made common class for similar property.